R/plot.R

```diff
@@ -1,13 +1,28 @@
-#' Simple plot an eeg_lst object.
-#'
-#' This function first downsamples the egg_lst and then converts it into a long tibble. See [as_tibble.eeg_lst] for details.
+#' Create a basic signal plot
+#' 
+#' \code{plot} creates a ggplot object in which the EEG signal over the whole 
```

Comment:
you can also do `plot` instead of \code{plot}. I'll replace it myself, but for the next docs.